### PR TITLE
What is a miner, and related articles revisions

### PR DESCRIPTION
**Pull request by MishaBoar**
What is a miner article, first iteration. I had to make some modifications to "What is a blockchain" and "What is a node", also to allow cross-linking.

Very difficult to keep an ELI5 tone in this one (even though the process of simplification took really forever on this one), even though the first two paragraphs could be enough for a person not interested at going more in depth.

### CODE REVIEW

**Comment by MishaBoar** [comment]
Expanding and simplifying a bit after a good night's sleep - please wait a bit to review.

**Comment by MishaBoar** [comment]
OK @RichieJohn - I submitted the revised version, which includes a simplified ELI5 section, fixes some typos, and adds some cross links.

**Comment by MishaBoar** [comment]
Added also article what-is-a-mining-pool.md for review.

File: content/dogepedia/articles/what-is-a-miner.md [hunk 1]
```diff
@@ -5,4 +5,62 @@ date = "2021-10-22"
...
+
+When a miner solves one of these puzzles before any other miner in the network, it is allowed to create a new block in the blockchain, thus confirming the transactions it has included in it. Miners make the blockchain secure and are rewarded for their work with so-called block rewards, consisting of 10,000 Dogecoin per mined block. Miners are also rewarded with the transaction fees associated with the transactions they have included into the block.
+
+If you want to explore this process more in-depth, you can proceed to the next session - but this should be enough to understand what a miner is and what it does for the Dogecoin network.
+
+### The Role of Miners in the Dogecoin Network {#the-role-of-miners}
+
+Miners play several fundamental roles.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-miner.md:22]: Delete full stop "." and insert colon ":" after the word "roles"

File: content/dogepedia/articles/what-is-a-miner.md [hunk 2]
```diff
@@ -5,4 +5,62 @@ date = "2021-10-22"
...
 +++
 
-A fairly accessible introduction to what a miner is, and the role of miners in the network.
+A **Miner** is a specialized [node](/dogepedia/how-tos/what-is-a-node/) in the Dogecoin network.
+
+After regular [nodes](/dogepedia/articles/what-is-a-node#full-nodes) in the network, by referencing the blockchain, have verified that a Dogecoin transaction is valid, the transaction is shown in a "pending" state. It stays in this state until a **miner** or a collective of miners known as a [**mining pool**](/dogepedia/articles/what-is-a-mining-pool) picks up the transaction.
+
+### Miners: ELI5 {#miners-eli5}
+
+Miners are nodes using special hardware (normally Scrypt ASIC miners) which allows them to solve cryptographic puzzles, in a process referred to as "mining". A miner receives new transactions from the other nodes in the network, combines them in a **block** following predetermined and inviolable rules enforced by the network, and then attempts to solve a puzzle that will generate a special number, called a hash, that is derived by combining the content of the block and a number called nonce.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-miner.md:14]: **bold** the word "nonce" and add in parentheses: "(_number used once_)". 

^ Suggest we do this here, and not below, because this is the first occasion in which the term is used.

File: content/dogepedia/articles/what-is-a-miner.md [hunk 3]
```diff
@@ -5,4 +5,62 @@ date = "2021-10-22"
...
+
+### The Role of Miners in the Dogecoin Network {#the-role-of-miners}
+
+Miners play several fundamental roles.
+
+- They confirm transactions by adding them into a new block
+- They secure the blockchain
+- They introduce new coins into circulation
+
+All the miners in the Dogecoin network are in constant competition to solve computationally-intensive puzzles, consisting in finding a special number called a **nonce** (*number used once*). The first miner - or the first mining pool - that is able to find a nonce that, when *hashed* along with the content of the block header, returns a number smaller than the current network's difficulty target, is the one allowed to write a new block into the blockchain.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-miner.md:28]: unbold "nonce", remove comment in parenthesis (as per above comment)

File: content/dogepedia/articles/what-is-a-mining-pool.md [hunk 4]
```diff
@@ -5,4 +5,18 @@ date = "2021-10-22"
   name = "Dogecoin"
 +++
 
-What is a mining pool, and does it relate to individual miners?
+Mining pools are services, operated by third parties not affiliated with Dogecoin, allowing group of [**miners**](/dogepedia/articles/what-is-a-miner/) to combine their [**hashing power**](/dogepedia/articles/what-is-a-miner/#hashing-power) when attempting to solve the puzzles involved in mining nodes.
```
Comment (RichieJohn) [inline content/dogepedia/articles/what-is-a-mining-pool.md:8]: delete "allowing" and add "which allow a"

**Review by RichieJohn — COMMENTED** [review]
Just a few minor suggestions for clarity, etc.

**Review by RichieJohn — APPROVED** [review]
